lib/csharp-models-to-json/EnumCollector.cs

```diff
@@ -20,9 +20,14 @@ public override void VisitEnumDeclaration(EnumDeclarationSyntax node)
             var values = new Dictionary<string, object>();
 
             foreach (var member in node.Members) {
-                values[member.Identifier.ToString()] = member.EqualsValue != null
+                var value = member.EqualsValue != null
                     ? member.EqualsValue.Value.ToString()
                     : null;
+
+                if (value?.StartsWith("0b") == true)
```

Comment:
Numbers can have underscores in C# and it doesn't have to start with "0b".   

_For example:_   
This is a valid C# enum:
```cs
public enum A
{
    A = 1,              // decimal: 1
    B = 1_002,          // decimal: 1002
    C = 0b011,          // binary: 3 in decimal
    D = 0b_100,         // binary: 4 in decimal
    E = 0x005,          // hexadecimal: 5 in decimal
    F = 0x000_01a,      // hexadecimal: 26 in decimal
    G                   // 27 in decimal
}
```
And the acceptable/ideal TS output has to be:
```ts
export enum A {
    A = 1,
    B = 1002,
    C = 0b011,
    D = 0b100,
    E = 0x005,
    F = 0x00001a,
    G = 27 // I think, currently this will print as 6 (incorrect)
}
```
If we can do this; that would be the great. :)

Reply:
So, your suggestion is to always remove the undescore. Not only in binary. That's right?

Reply:
Yes, I think so, given the `member.EqualsValue.Value` is always a numerical value. 
I tested this on my project too and it is working.